### PR TITLE
Use correct repo name for easyblocks

### PR DESCRIPTION
**Pull request by ocaisa**

### CODE REVIEW

**Review by SebastianAchilles — APPROVED** [review]
lgtm

File: eb_from_pr_upload_generoso.sh [hunk 1]
```diff
@@ -36,7 +36,7 @@ export INTEL_LICENSE_FILE=${TOPDIR}/maintainers/licenses/intel.lic
 module use ${EASYBUILD_PREFIX}/modules/all
 
 repo_pr_arg='--from-pr'
-if [ "EB_REPO" == "easyblocks" ]; then
+if [ "EB_REPO" == "easybuild-easyblocks" ]; then
```
Comment (SebastianAchilles) [inline eb_from_pr_upload_generoso.sh:39]:
```suggestion
if [ $EB_REPO == "easybuild-easyblocks" ]; then
```

File: eb_from_pr_upload_jsc-zen2.sh [hunk 2]
```diff
@@ -37,7 +37,7 @@ export EASYBUILD_SET_GID_BIT=1
 export EASYBUILD_UMASK='022'
 
 repo_pr_arg='--from-pr'
-if [ "EB_REPO" == "easyblocks" ]; then
+if [ "EB_REPO" == "easybuild-easyblocks" ]; then
```
Comment (SebastianAchilles) [inline eb_from_pr_upload_jsc-zen2.sh:40]:
```suggestion
if [ $EB_REPO == "easybuild-easyblocks" ]; then
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]

**Comment by ocaisa** [comment]
Thanks for the fixes @SebastianAchilles

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
deployed on both `jsc-zen2` and `generoso`

**Pull request merged**